models/aether-2.0.x/go.mod

```diff
@@ -3,17 +3,15 @@ module github.com/onosproject/aether-models/models/aether-2.0.x/v2
 go 1.16
 
 require (
-	github.com/SeanCondon/xpath v0.0.0-20220225125014-419de017925c
-	github.com/atomix/atomix-go-framework v0.10.1 // indirect
+	github.com/SeanCondon/xpath v0.0.0-20220628084621-97cfdefbc266
 	github.com/ghodss/yaml v1.0.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
-	github.com/onosproject/config-models v0.10.16
+	github.com/onosproject/config-models v0.10.23
 	github.com/onosproject/onos-api/go v0.9.14
```

Comment:
Why the config-model version for the models are v0.10.23? The latest released version is v0.10.24